ui/component/claimPreview/view.jsx

```diff
@@ -244,16 +252,22 @@ const ClaimPreview = forwardRef<any, {}>((props: Props, ref: any) => {
   }
 
   const handleNavLinkClick = (e) => {
-    if (onClick) {
-      onClick(e, claim, indexInContainer); // not sure indexInContainer is used for anything.
+    if (playItemsOnClick) {
+      e.preventDefault();
+      e.stopPropagation();
+      doUriInitiatePlay({ uri, collectionId, source: collectionId === 'queue' ? collectionId : undefined }, true, true);
+    } else {
+      if (onClick) {
+        onClick(e, claim, indexInContainer); // not sure indexInContainer is used for anything.
+      }
+      e.stopPropagation();
     }
-    e.stopPropagation();
   };
```

Comment:
--- Not a blocker, but consider in future ---
From a shared component design perspective, I think we should try to make the component as dumb as possible, only doing what's necessary to display the data. Specific actions like `doUriInitiatePlay` shouldn't be a `ClaimPreview`'s responsibility -- we can provide generic hooks and render functions, etc. to tie up these actions.

- Is it possible for the client (`playlistCard`) to define `ClaimList :: onClick` instead, and do the `e.preventDefault` and `doUriInitiatePlay` there?
  - I believe `onClick` is already propagated between `ClaimList` and `ClaimPreview`, so the drilling is already done.  And you should have all the collection params at the client side.  The only thing to figure how is if the tweaks in `navLinkProps` can have an alternative solution.
- We would then also not need `playItemsOnClick`

If `doUriInitiatePlay` is a very common action, then incorperating it would make sense, but probably not when it's for a very specific use-case.  It makes tweaking `handleNavLinkClick` harder in the future.